### PR TITLE
MUSARK-957 - Feature/current location api changes

### PR DESCRIPTION
**Pull request by ingarabr**
Depending on https://github.com/MUSIT-Norway/musit/pull/286

- Update the api call to support the changes done to the backend.
- Add flow type to config.js
- add coverage script.

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -8,8 +11,8 @@ export default {
   useDevTools: process.env.DEV_TOOLS || false,
   print: {
     labelConfig: {
-      codeFormat: { 4: 2 },
-      canSelectPath: { 4: true, 5: true }
+      codeFormat: { '4': 2 },
```
Comment (jarlah) [inline src/config.js:14]: Have you done is as a result of getting flow type errors? Or is this an actual improvement?

**Review by jarlah — COMMENTED** [review]
Just a small question

File: src/config.js (hunk 1 above)
Comment (ingarabr) [reply src/config.js:14]: It was flow that complained about it.

Comment (jarlah) [reply src/config.js:14]: as long as it works as it did before, I am willing to let you a say a monkey did it ;)

**Review by jarlah — APPROVED** [review]

**Pull request merged**